acestep/core/generation/handler/lyric_score.py

```
@@ -0,0 +1,164 @@
+                aligner=aligner,
+                matrix=pure_matrix_dit,
+                pure_lyric_ids=pure_lyric_ids,
+                custom_layers_config=custom_layers_config,
+            )
+            return {
+                "lm_score": lm_score,
+                "dit_score": dit_score,
+                "success": True,
+                "error": None,
+            }
+        except Exception as exc:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)